src/edu/psu/sweng500/team8/gui/GridPanel.java

```diff
@@ -305,6 +309,10 @@ private void markGivenCell(Cell cell) {
 		HighlightPainter painter = new DefaultHighlighter.DefaultHighlightPainter(
 				Color.green);
 
+		/* TODO - THIS MAY NOT BE THE BEST PLACE TO HAVE THIS SET BUT I NEED THIS TO MAKE PENCIL MARK WORKING
+		 * DO NOT FORGET REVISIT AND UPDATE ValueType logic */
+		cell.setType(ValueType.Given);
```

Comment:
please test this. when I was running and checking cell.getType, Given was not being returned for the pre-populated cell. That's why I added the comment. I tested again and it's the same thing.
I am going to leave it there until it's re-re-verified.